database/schema.sql

```diff
@@ -4,10 +4,17 @@
 -- fixed size.
 -- `u32`s are stored in `bigint` which is an 8 bytes signed integer because Postgre does not have
 -- unsigned integers.
+BEGIN;
 
-CREATE TYPE OrderKind AS ENUM ('buy', 'sell');
+DO $$
+BEGIN
```

Comment:
Why is this `DO $$` needed? The reference I found is https://www.postgresql.org/docs/current/sql-do.html but not sure what is going on here?

Reply:
It's [dollar quoting](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-DOLLAR-QUOTING) to avoid conflict with any quotes that happen inside the query.